model-optimizer/extensions/front/tf/placeholder_ext.py

```diff
@@ -13,9 +15,30 @@ class PlaceholderFrontExtractor(FrontExtractorOp):
 
     @classmethod
     def extract(cls, node):
+        shape = shape_array([])
+        # Extract output shape from `shape` attribute
+        extracted_shape = tf_tensor_shape(node.pb.attr["shape"].shape)
+        if len(extracted_shape) != 0 and not node.pb.attr["shape"].shape.unknown_rank:
+            shape = extracted_shape
+
+        # Extract output shape from `_output_shapes` attribute if it is possible
+        extracted_output_shapes = node.pb.attr["_output_shapes"].list.shape
+        if len(extracted_output_shapes) > 0:
```

Comment:
Could you check if there can be `if len(extracted_output_shapes) != 1` condition?
Details have been discussed offline.

Reply:
Checked, it can be `if len(extracted_output_shapes) == 1`, change applyed.